lib/twilio.rb

```diff
@@ -0,0 +1,13 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class Twilio_messenger
```

Comment:
Style/ClassAndModuleCamelCase: Use CamelCase for classes and modules. (https://github.com/bbatsov/ruby-style-guide#camelcase-classes)